tests/system/src/test/java/test/com/sun/javafx/application/InitializeJavaFXLaunchTest.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2012, 2021, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Minor year correction-> `Copyright (c) 2021,`

Reply:
Done, so it's the year where changes happened, ok!

Reply:
In general, the pattern is: `year_created, year_last_modified,` (with the last modified year omitted if it's the same as the creation year).